clock/config/database.yml

```diff
@@ -81,5 +81,5 @@ test:
 production:
   <<: *default
   database: clock_production
-  username: clock
-  password: <%= ENV['CLOCK_DATABASE_PASSWORD'] %>
```

Comment:
You can use the dot-env gem too for setting local env vars 😄 
https://github.com/bkeepers/dotenv